spec/misc.spec.js

```diff
@@ -12,4 +15,39 @@ describe('Run misc neu commands', () => {
             assert.ok(output.data.includes('Usage: neu [options] [command]'));
         });
     });
+    describe('Test neu version command', () => {
+        it('returns output of neu version --help', async() => {
+            let output = runner.run('neu version --help');
+
+            assert.equal(output.error, null);
+            assert.equal(output.status, 0);
+            assert.ok(typeof output.data == 'string');
+            assert.ok(output.data.includes('Usage: neu version [options]'));
+        });
+        it('returns neu version installed globally', async() => {
+            let output = runner.run('neu version');
+
+            assert.equal(output.error, null);
+            assert.equal(output.status, 0);
+            assert.ok(typeof output.data == 'string');
+            assert.ok(output.data.includes('Global') &&
+            output.data.includes('neu CLI: v'));
+        });
+        it('returns project specific neu version details', async() => {
+            let output = runner.run('cd test-app && neu version');
+
+            assert.equal(output.error, null);
+            assert.equal(output.status, 0);
+            assert.ok(typeof output.data == 'string');
+            assert.ok(output.data.includes('Global') &&
+                output.data.includes('neu CLI: v'));
+            assert.ok(output.data.includes('Project') &&
+                output.data.includes('Neutralinojs binaries') &&
+                output.data.includes('Neutralinojs client') &&
+                output.data.includes('Project version'));
+        });
+    });
+    after(() => {
+        runner.run('rm -r test*');
```

Comment:
All looks great, but I have an idea to make this function call better. I think we can have the clean-up code in `runner.cleanup()` with Node.js functions (not Unix command-line like this). We can merge this after fixing this ;). I think we can use `.run()` only for neu command-lines.

Thanks :tada: